src/skmultiflow/_demos/_test_mol.py

```diff
@@ -23,7 +23,8 @@ def demo():
     logging.basicConfig(format='%(message)s', level=logging.INFO)
 
     # Setup the file stream
-    stream = FileStream("../data/datasets/music.csv", 0, 6)
+    stream = FileStream("https://raw.githubusercontent.com/scikit-multiflow/streaming-datasets/"
+                        "master/", 0, 6)
```

Comment:
```suggestion
    stream = FileStream("https://raw.githubusercontent.com/scikit-multiflow/streaming-datasets/"
                        "master/music.csv", 0, 6)
```

Reply:
Fixed in e8b8f3fd3bb60bd1e6ad657e3fe45da11ac0a8d2